_includes/manuals/1.9/7.3_getting_help.md

```diff
@@ -8,3 +8,12 @@ Mailing lists are available via Google Groups. Much like IRC, we have a general
 
 * [foreman-users](https://groups.google.com/forum/?fromgroups#!forum/foreman-users)
 * [foreman-dev](https://groups.google.com/forum/?fromgroups#!forum/foreman-dev)
+
+### Gathering information
+* In order to troubleshoot and get relevant data use Foreman-debug specified in [genral support] (http://theforeman.org/support.html)
```

Comment:
Remove the bullet points, these are sentences.  "foreman-debug" is a command, so is case sensitive - lower case it.  "general" is misspelt, and the link isn't working - you have a space between the brackets.  The link should also not contain the domain, just /support.html.